testlib/src/main/java/io/spine/testing/logging/LoggerTest.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2019, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.testing.logging;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Abstract base for tests of logging.
+ */
+public abstract class LoggerTest {
+
+    /** The class which performs the log operations. */
+    private final Class<?> loggingClass;
+    /** The value of the property used by the logger before the test. */
+    private boolean useParentHandler;
+    /** The value the logger has before the tests. */
+    private final Level previousLevel;
+    /** The level to be used during the tests. */
+    private final Level level;
+    /** The handler which remembers log records and perform assertions. */
+    private @Nullable AssertingHandler handler;
+
+    /**
+     * Creates new test suite.
+     *
+     * @param loggingClass the class which performs the logging operations
+     * @param level        the level of logging we are interested in the tests
```

Comment:
`we are interested in in the tests`

Reply:
I don't understand this comment. Please explain.

Reply:
One can be [interested in](https://ell.stackexchange.com/questions/32998/interested-in-to-whats-the-difference) smth. So in our case, we are interested in the log level that is used in the tests, right?

Reply:
Now I got it. I've changed the language so that we avoid double “in”.